src/updatechecker.cpp

```diff
@@ -228,7 +228,7 @@ void UpdateChecker::PerformUpdateCheck()
         CheckForInsecureURL(url, "appcast feed");
 
         StringDownloadSink appcast_xml;
-        DownloadFile(url, &appcast_xml, this, Download_BypassProxies);
+        DownloadFile(url, &appcast_xml, this, WideToAnsi(Settings::GetHttpHeaderString()), Download_BypassProxies);
```

Comment:
What's the point of using wide strings for the API and storage, when you're going to throw it all away (without even a valid conversion to something like UTF-8)? It doesn't make sense for HTTP headers anyway.